Bottom: b44307b2cded520265a8471358f96f90caf938bb
Top:    684605d0e999cb7ce8a212e52f808b699f82f939
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-07-24 17:44:29 +0100

hid/gtk: Fixup Zoom action.

If the passed coordinates are both zero, we pass the PCB coordinates
of the board center into a function which currently expects screen
coordinates. FAIL.

Remove that special case code - it should just work without it.


---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index b11d32c..f0d3da5 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -2382,7 +2382,6 @@ ghid_parse_arguments (int *argc, char ***argv)
   gtk_init (argc, argv);
 
   gport = &ghid_port;
-  gport->zoom = 300.0;
   pixel_slop = 300;
 
   ghid_init_renderer (argc, argv, gport);
